criticality_score/generate.py

```diff
@@ -111,7 +110,16 @@ def main():
     repo_urls = set()
     for rnd in range(1, 4):
         print(f'Finding repos (round {rnd}):')
-        repo_urls.update(get_repo_urls(args.language, args.sample_size))
+        urls = []
+        if (args.language):
```

Comment:
Can you move lines 114-120 into a helper function get_github_repo_urls(languages), will be adding gitlab in future, so moving this into a helper is better.

Reply:
Okay, it's now like this:

```
repo_urls.update(get_github_repo_urls(args.sample_size, args.language))

def get_github_repo_urls(sample_size, languages):
    urls = []
    if (languages):
        for lang in languages:
            lang = lang.lower()
            for github_lang in LANGUAGE_SEARCH_MAP.get(lang, lang):
                urls = get_github_repo_urls_for_language(urls, sample_size, github_lang)
    else:
        urls = get_github_repo_urls_for_language(urls, sample_size)

    return urls
```